_rules/__tests__/headings.js

```diff
@@ -22,14 +22,36 @@ const getAllHeadingsFromMarkdownBody = body => {
 	}, [])
 }
 
+/**
+ * Extract a number from a given string
+ * @param {String} str given string
+ * @returns {Number}
+ */
 const extractNumberFromGivenString = (str) => {
-	return str.match(/\d+/)[0]
+	const match = str.match(/\d+/)[0]
+	return parseInt(match)
 }
 
+/**
+ * Check if a given array has duplicates
+ * @param {Array<Object>} array arr
+ * @returns {Boolean}
+ */
 const arrayHasDuplicates = (array) => {
 	return (new Set(array)).size !== array.length;
 }
 
+/**
+ * Check if a given array of numbers is in ascending order
+ * @param {Array<Number>} arr array of numbers to verify if they are ascending
+ * @returns {Boolean}
+ */
+const isAscending = arr => {
```

Comment:
given that we want not only ascending order but also start at 1, it should work with:
`arr.every((elt, idx) => elt === idx+1)`
which is faster and easier to read.